xprocess.py

```diff
@@ -47,17 +47,20 @@ def terminate(self, **kwargs):
         if not self.pid or not self.isrunning():
```

Comment:
(not related to this line, but GH only let's me comment on the diff)

I only now noticed "setting ``kill_proc_tree`` flag", but I believe a more natural wording would be "setting ``kill_proc_tree`` keyword-only parameter".

Reply:
Done